### PR TITLE
bump nuraft

### PR DESCRIPTION
**Pull request by raakella1**

### CODE REVIEW

**Comment by yamingk** [comment]
@raakella1 need to include the header file for build to pass, right?

**Comment by raakella1** [comment]
> @raakella1 need to include the header file for build to pass, right?

Yes, this is still a draft.

**Comment by yamingk** [comment]
it is good this PR build could pass, interestingly other PRs hit the similar issue to seem to be hang in 1st flavor

**Review by yamingk — APPROVED** [review]

File: .github/workflows/build_dependencies.yml [hunk 1]
```diff
@@ -211,10 +211,10 @@ jobs:
         sanitize=$([[ "${{ inputs.tooling }}" == "Sanitize" ]] && echo "True" || echo "False")
         pre=$([[ "${{ inputs.build-type }}" != "Debug" ]] && echo "-o sisl:prerelease=${{ inputs.prerelease }}" || echo "")
         conan create \
+            -c tools.build:skip_test=True \
```
Comment (yamingk) [inline .github/workflows/build_dependencies.yml:214]: this will skip testing for HomeStore testing binaries as well, is this expected?

**Pull request merged**